test/creating-settings.js

```diff
@@ -207,22 +223,6 @@ describe('Creating throttling settings', () => {
       });
     });
   });
-
-  describe('when a http endpoint is defined in shorthand', () => {
```

Comment:
This test is needed because events can also be defined in the `serverless.yml` like this:

```
events:
  - http: get /items
```

The plugin should support this as well.